go.mod

```diff
@@ -49,7 +51,7 @@ require (
 	github.com/bytedance/sonic v1.9.1 // indirect
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
 	github.com/chenzhuoyu/base64x v0.0.0-20221115062448-fe3a3abad311 // indirect
-	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc // indirect
```

Comment:
i find this strange as the latest is 1.1.1? https://pkg.go.dev/github.com/davecgh/go-spew/spew?tab=versions
https://github.com/davecgh/go-spew and the additional commit is irrelevant

Reply:
yeah, let's leave it